Makefile

```diff
@@ -160,7 +160,7 @@ pytest-integration:
 .PHONY: mypy
 # Run Mypy static type checker.
 mypy:
-	./scripts/mypy_custom_script.py
+	mypy --config-file=lib/mypy.ini --namespace-packages lib/streamlit/ lib/tests/streamlit/typing/ scripts/ e2e/scripts/
```

Comment:
nit:  I think you can leave out ` e2e/scripts/`. We are almost done with migrating all cypress e2e scripts. We might want to add `e2e_playwright` here instead, but this might require a bit more time to fix typing issues.

Reply:
As described, seems to me like you should remove `e2e/scripts/` when you finish that migration, then; but I'm not really familiar with the situation, so I'll trust your judgement and make the change 👍.

I tried putting e2e_playwright in as well, since you mentioned it, and there were like a hundred errors in there, so not today.

Reply:
With the migration PRs open, we will be down to two remaining scripts. Removing it now ensures that it won't stick around as a historical artifact. But either way is probably fine.